docs/wallet/universal-links.md

```diff
@@ -269,19 +269,32 @@ const finishUrl = encodeURIComponent("https://my-app.org/finish");
 // Base64 encode the verification request
 const base64EncodedRequest = btoa(JSON.stringify(request));
 
-// URL encode the verification request with the parameters
-const encodedRequestWithParams =(`i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`);
 
-// Open the Web Wallet URL with the encoded request
-window.open(`https://wallet-dev.privado.id/#${encodedRequestWithParams}`);
+// Configure the Wallet URL(universal link)
```

Comment:
missing space

> `URL(uni`

Reply:
Also, why removing `Web`? We should always refer to `Web Wallet`

Reply:
Fixed

Reply:
Ok. I guess...